src/server/queries/recipeView.ts

```
@@ -0,0 +1,495 @@
+  // Use the first applicable rule
+  const rule = scalingRules[0];
+  if (!rule) return baseQuantity * servingsMultiplier;
+
+  switch (rule.ruleType) {
+    case "fixed":
+      // Quantity stays constant regardless of servings
+      return baseQuantity;
+
+    case "step":
+      // Round up to the nearest step size
+      const stepSize = rule.stepSize ? parseFloat(rule.stepSize) : 1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)